web/app/js/index.js

```diff
@@ -36,18 +36,23 @@ import { dashboardTheme } from './components/util/theme.js';
 const appMain = document.getElementById('main');
 const appData = !appMain ? {} : appMain.dataset;
 
-let pathPrefix = '';
+let pathPrefix = appData.pathPrefix;
 const proxyPathMatch = window.location.pathname.match(/\/api\/v1\/namespaces\/.*\/proxy/g);
 if (proxyPathMatch) {
   pathPrefix = proxyPathMatch[0];
 }
 
+// Make sure the pathPrefix does not end with a '/'
+if (pathPrefix.slice(-1) === '/') {
+  pathPrefix = pathPrefix.slice(0, -1);
+}
+
 let defaultNamespace = 'default';
-const pathArray = window.location.pathname.split('/');
+const pathArray = window.location.pathname.substring(pathPrefix.length).split('/');
```

Comment:
It feels a little weird to me to not make sure that the path really does start with `pathPrefix` before doing this substring...